### ISSUE TITLE
[Bug] qwen3-235b-a22b-instruct-2507 lora 推理出现Exception: No corresponding LoRA layer supported for <class 'sglang.srt.layers.linear.ReplicatedLinear'>.

### ISSUE THREAD

**Issue by artless-spirit** [sgl-project/sglang#8903]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sglang/discussions/new/choose Otherwise, it will be closed.
- [x] 5. Please use English, otherwise it will be closed.

### Describe the bug

lora训练的qwen3-235b-a22b-instruct-2507，在使用lora serving推理的时候，出现Exception: No corresponding LoRA layer supported for <class 'sglang.srt.layers.linear.ReplicatedLinear'>.

### Reproduction

 CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python3 -m sglang.launch_server --model-path /pretrained_models/qwen3-235b-a22b-instruct-2507/ --tp 8 --host=0.0.0.0 --port 80 --trust-remote-code --json-model-override-args '{"rope_scaling":{"rope_type":"yarn","factor":4.0,"original_max_position_embeddings":32768}}' --served-model-name qwen3-07041239-lora --lora-paths lora0=/finetuned_models/checkpoint-100/  --max-loras-per-batch 1 --lora-backend triton --disable-radix-cache
Package                   Version
------------------------- -------------
aiohappyeyeballs          2.6.1
aiohttp                   3.12.15
aiosignal                 1.4.0
airportsdata              20250706
annotated-types           0.7.0
anthropic                 0.61.0
anyio                     4.10.0
asttokens                 3.0.0
attrs                     25.3.0
blinker                   1.7.0
blobfile                  3.0.0
build                     1.3.0
certifi                   2025.8.3
cffi                      1.17.1
charset-normalizer        3.4.2
click                     8.2.1
cloudpickle               3.1.1
compressed-tensors        0.10.2
cryptography              41.0.7
cuda-bindings             13.0.0
cuda-pathfinder           1.1.0
cuda-python               13.0.0
datasets                  4.0.0
dbus-python               1.3.2
decorator                 5.2.1
decord                    0.6.0
dill                      0.3.8
diskcache                 5.6.3
distro                    1.9.0
distro-info               1.7+build1
einops                    0.8.1
executing                 2.2.0
fastapi                   0.116.1
filelock                  3.18.0
flashinfer-python         0.2.9rc2
frozenlist                1.7.0
fsspec                    2025.3.0
h11                       0.16.0
hf_transfer               0.1.9
hf-xet                    1.1.7
httpcore                  1.0.9
httplib2                  0.20.4
httpx                     0.28.1
huggingface-hub           0.34.3
idna                      3.10
importlib_metadata        8.7.0
interegular               0.3.3
ipython                   9.4.0
ipython_pygments_lexers   1.1.1
jedi                      0.19.2
Jinja2                    3.1.6
jiter                     0.10.0
jsonschema                4.25.0
jsonschema-specifications 2025.4.1
lark                      1.2.2
launchpadlib              1.11.0
lazr.restfulclient        0.14.6
lazr.uri                  1.0.6
litellm                   1.75.0
llguidance                0.7.30
lxml                      6.0.0
MarkupSafe                3.0.2
matplotlib-inline         0.1.7
modelscope                1.28.2
mpmath                    1.3.0
msgspec                   0.19.0
multidict                 6.6.3
multiprocess              0.70.16
nest-asyncio              1.6.0
networkx                  3.5
ninja                     1.11.1.4
numpy                     2.3.2
nvidia-cublas-cu12        12.6.4.1
nvidia-cuda-cupti-cu12    12.6.80
nvidia-cuda-nvrtc-cu12    12.6.77
nvidia-cuda-runtime-cu12  12.6.77
nvidia-cudnn-cu12         9.5.1.17
nvidia-cudnn-frontend     1.13.0
nvidia-cufft-cu12         11.3.0.4
nvidia-cufile-cu12        1.11.1.6
nvidia-curand-cu12        10.3.7.77
nvidia-cusolver-cu12      11.7.1.2
nvidia-cusparse-cu12      12.5.4.2
nvidia-cusparselt-cu12    0.6.3
nvidia-ml-py              12.575.51
nvidia-nccl-cu12          2.26.2
nvidia-nvjitlink-cu12     12.6.85
nvidia-nvshmem-cu12       3.3.20
nvidia-nvtx-cu12          12.6.77
oauthlib                  3.2.2
openai                    1.99.1
orjson                    3.11.1
outlines                  0.1.11
outlines_core             0.1.26
packaging                 25.0
pandas                    2.3.1
parso                     0.8.4
partial-json-parser       0.2.1.1.post6
pexpect                   4.9.0
pillow                    11.3.0
pip                       24.0
prometheus_client         0.22.1
prompt_toolkit            3.0.51
propcache                 0.3.2
psutil                    7.0.0
ptyprocess                0.7.0
pure_eval                 0.2.3
pyarrow                   21.0.0
pybase64                  1.4.2
pycountry                 24.6.1
pycparser                 2.22
pycryptodomex             3.23.0
pydantic                  2.11.7
pydantic_core             2.33.2
Pygments                  2.19.2
PyGObject                 3.48.2
PyJWT                     2.7.0
pynvml                    12.0.0
pyparsing                 3.1.1
pyproject_hooks           1.2.0
python-apt                2.7.7+ubuntu4
python-dateutil           2.9.0.post0
python-dotenv             1.1.1
python-multipart          0.0.20
pytz                      2025.2
PyYAML                    6.0.2
pyzmq                     27.0.1
referencing               0.36.2
regex                     2025.7.33
requests                  2.32.4
rpds-py                   0.26.0
safetensors               0.6.1
scipy                     1.16.1
sentencepiece             0.2.0
setproctitle              1.3.6
setuptools                68.1.2
sgl-kernel                0.2.8
sglang                    0.4.10.post2
six                       1.16.0
sniffio                   1.3.1
soundfile                 0.13.1
stack-data                0.6.3
starlette                 0.47.2
sympy                     1.14.0
tiktoken                  0.10.0
timm                      1.0.16
tokenizers                0.21.4
torch                     2.7.1
torch_memory_saver        0.0.8
torchao                   0.9.0
torchaudio                2.7.1
torchvision               0.22.1
tqdm                      4.67.1
traitlets                 5.14.3
transformers              4.54.1
triton                    3.3.1
typing_extensions         4.14.1
typing-inspection         0.4.1
tzdata                    2025.2
unattended-upgrades       0.1
urllib3                   2.5.0
uvicorn                   0.35.0
uvloop                    0.21.0
wadllib                   1.3.6
wcwidth                   0.2.13
wheel                     0.42.0
xgrammar                  0.1.22
xxhash                    3.5.0
yarl                      1.20.1
zipp                      3.23.0

### Environment

ubuntu 22.04 cuda12.4
